tests/regression/issue2172.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+# bats file_tags=regression,bin:init
+
+setup() {
+  load "../common/lib"
+  CK8S_CONFIG_PATH="$(mktemp --directory)"
+  export CK8S_CONFIG_PATH
+  export CK8S_ENVIRONMENT_NAME="issue2172-regression"
+  export CK8S_CLOUD_PROVIDER="baremetal"
+  export CK8S_FLAVOR="dev"
+  common_setup
+
+}
```

Comment:
Did you consider using some of the [env helper functions](https://github.com/elastisys/compliantkubernetes-apps/blob/2308b7dc6f4ccf16ba37dc819c3c55c0da43d60f/tests/common/lib/env.bash#L1)? If you did, what was preventing you from using them?

Reply:
I was either unaware of those or thought they could not be used in this context, perhaps because I looked at [tests/regression/template.bats](https://github.com/elastisys/compliantkubernetes-apps/blob/2308b7dc6f4ccf16ba37dc819c3c55c0da43d60f/tests/regression/template.bats)